MapboxNavigation/RouteMapViewController.swift

```diff
@@ -98,37 +102,41 @@ class RouteMapViewController: UIViewController {
     var annotatesSpokenInstructions = false
     
     var overheadInsets: UIEdgeInsets {
-        return UIEdgeInsets(top: instructionsBannerView.bounds.height, left: 20, bottom: bottomBannerView?.bounds.height ?? 40, right: 20)
+        return UIEdgeInsets(top: navigationView.instructionsBannerView.bounds.height, left: 20, bottom: navigationView.bottomBannerView.bounds.height, right: 20)
+    }
+    
+
+    convenience init(routeController: RouteController, delegate: RouteMapViewControllerDelegate? = nil) {
+        self.init()
+        self.routeController = routeController
+        self.delegate = delegate
+    }
+    
+    
+    override func loadView() {
```

Comment:
should we call `super.loadView()` here ?

Reply:
[Nope.](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621454-loadview)